package.json

```diff
@@ -4,21 +4,22 @@
   "description": "It's cool when users own their content.",
   "main": "index.js",
   "scripts": {
-    "build": "./node_modules/.bin/gulp",
+    "build": "gulp",
```

Comment:
Why change these? They are deliberately referencing the local install of gulp (and later mocha) in case the person's global install is the wrong version. Either way works for docker, but not for a normal git clone & npm install.